src/operator/quantization/quantized_conv.cu

```diff
@@ -110,6 +110,9 @@ class QuantizedCuDNNConvOp {
     const TShape& fshape = filter.shape_;
     const TShape& oshape = out.shape_;
 
+    CHECK_EQ(data.type_flag_, mshadow::kInt8)
+      << "currently, uint8 quantization is only supported by CPU, "
+         "please switch to the context of CPU or int8 data type for GPU.";
```

Comment:
can we add inside quantize-inl.h, this way it will return an error message even for networks without this op.

Reply:
ok, added:)